include/aspect/mesh_deformation/fastscape.h

```diff
@@ -86,6 +87,18 @@ namespace aspect
          */
         void parse_parameters (ParameterHandler &prm) override;
 
+        /**
+         * Enumeration for selecting which type of additional ouput to use in Fastscape vtk.
+         * Select between Fastscape variables.
+         */
+        enum FSScheme
+        {
+          kf,
+          kd,
+          uplift
+        };
+        FSScheme additional_output;
```

Comment:
The variable name also does not quite reflect its purpose. Perhaps `additional_output_variables`?